packages/banner/src/react/index.tsx

```diff
@@ -64,19 +65,20 @@ const Banner = React.forwardRef((props, ref) => {
 
 Banner.displayName = 'Banner'
 
-interface ButtonProps
-  extends React.HTMLAttributes<HTMLButtonElement | HTMLAnchorElement> {}
-
-interface ButtonStatics {}
-
-interface ButtonComponent
-  extends RefForwardingComponent<
-    ButtonProps,
-    HTMLButtonElement | HTMLAnchorElement,
-    ButtonStatics
-  > {}
-
-const Button = React.forwardRef((props, forwardRef) => {
+interface AnchorProps extends HTMLPropsFor<'a'> {
+  href: string
+}
+interface ButtonProps extends HTMLPropsFor<'button'> {
+  href?: undefined
+}
+type ButtonComponent = React.ForwardRefExoticComponent<unknown> & {
```

Comment:
Are there other components outside of this PR that need their overloads updated to be like this?

Reply:
i think NavUser needs the `unknown`...but that's about it